### PR TITLE
docs: add profile for uestcer

### PR DESCRIPTION
**Pull request by suenyiyang**
I used HTML elements and inline CSS for showing the information in a different way, but it seems not work in my forked repo. I'd love to help improve the content if there is any need.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+
+</main>
+
+<main style="width: 30%">
+
+### Contribution
+
+- Share your great pieces to help others;
+- Use your inspiration and creativity to build up interesting sites and tools;
+- Invite more UESTCer to apply for this org.
```
Comment (Xunzhuo) [inline profile/README.md]: minor: UESTCers

File: profile/README.md [hunk 2]
```diff
@@ -1,12 +1,43 @@
+<center>
+
+![UESTC Logo](https://www.uestc.edu.cn/c13209856342b0df2c7ee4eaeb07db5a.png)
```
Comment (Xunzhuo) [inline profile/README.md:3]: Can this image to be aligned into center?

Comment (suenyiyang) [reply profile/README.md:3]: It seems the `center` element doesn't work, I'll try another way of making it center aligned.

Comment (suenyiyang) [reply profile/README.md:3]: > Can this image to be aligned into center?

I set the whole title part center aligned.

**Review by Xunzhuo — APPROVED** [review]

**Comment by Xunzhuo** [comment]
Thanks @syy11cn

**Pull request merged**